packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/aspnetcore/remotesignout.md

```diff
@@ -1,17 +1,5 @@
 ASP.NET Core hosts an internal sign-out handler at `/signout/callback`. You need to add it as a valid **Logout redirect URI** in Okta. See [Define the signout callback](/docs/guides/sign-users-out/define-signout-callback/) for more information on defining this URI in your application settings.
 
-Open your Okta Developer Console:
```

Comment:
Should we mention that after users sign out of Okta they will be redirected to the location defined in _Define your sign out callback_?